<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-UB42IN2" failures="0" tests="2" name="Default test" time="0.285" errors="0" timestamp="2024-11-17T16:48:39 IST">
  <testcase classname="Test.BaseTest" name="@BeforeMethod setup" time="0.019">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentReports.createTest(String)&amp;quot; because &amp;quot;comtmb.Reports.ExtentReport.extent&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "comtmb.Reports.ExtentReport.extent" is null
at comtmb.Reports.ExtentReport.createTest(ExtentReport.java:47)
at Test.BaseTest.setup(BaseTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:81)
at org.testng.internal.thread.Async$AsyncTask.run(Async.java:34)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase classname="Test.BaseTest" name="@AfterMethod teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod teardown -->
  <testcase classname="Test.BaseTest" name="@BeforeMethod setup" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase classname="Test.BaseTest" name="@AfterMethod teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod teardown -->
  <testcase classname="Test.OrangeHRMTestCases" name="loginLogoutTest" time="0.0">
    <skipped/>
  </testcase> <!-- loginLogoutTest -->
  <testcase classname="Test.OrangeHRMTestCases" name="newTest" time="0.0">
    <skipped/>
  </testcase> <!-- newTest -->
</testsuite> <!-- Default test -->
